web/components/issues/issue-layouts/empty-states/archived-issues.tsx

```diff
@@ -61,33 +38,21 @@ export const ProjectArchivedEmptyState: React.FC = observer(() => {
     });
   };
 
-  const isEditingAllowed = !!currentProjectRole && currentProjectRole >= EUserProjectRoles.MEMBER;
-
-  const emptyStateProps: EmptyStateProps =
-    issueFilterCount > 0
-      ? {
-          title: EMPTY_FILTER_STATE_DETAILS["archived"].title,
-          image: currentLayoutEmptyStateImagePath,
-          secondaryButton: {
-            text: EMPTY_FILTER_STATE_DETAILS["archived"].secondaryButton?.text,
-            onClick: handleClearAllFilters,
-          },
-        }
-      : {
-          title: EMPTY_ISSUE_STATE_DETAILS["archived"].title,
-          description: EMPTY_ISSUE_STATE_DETAILS["archived"].description,
-          image: EmptyStateImagePath,
-          primaryButton: {
-            text: EMPTY_ISSUE_STATE_DETAILS["archived"].primaryButton.text,
-            onClick: () => router.push(`/${workspaceSlug}/projects/${projectId}/settings/automations`),
-          },
-          size: "sm",
-          disabled: !isEditingAllowed,
-        };
+  const emptyStateType = issueFilterCount > 0 ? "project-archived-empty-filter" : "project-archived-no-issues";
+  const additionalPath = issueFilterCount > 0 ? activeLayout ?? "list" : undefined;
 
   return (
     <div className="relative h-full w-full overflow-y-auto">
-      <EmptyState {...emptyStateProps} />
+      <EmptyState
+        type={emptyStateType}
+        additionalPath={additionalPath}
+        primaryButtonOnClick={
+          issueFilterCount > 0
+            ? undefined
+            : () => router.push(`/${workspaceSlug}/projects/${projectId}/settings/automations`)
```

Comment:
This looks like it could be a controlLink so that user can also right click to open in new tab, please check